packages/ui/src/common/components/ListItemLoader.tsx

```diff
@@ -9,30 +9,36 @@ interface ListItemLoaderProps {
   count?: number
   height?: string
   id?: string
+  gap?: string
+  padding?: string
 }
 
-export const ListItemLoader = ({ children, count = 1, id, ...styleProps }: ListItemLoaderProps) => {
+export const ListItemLoader = ({ children, count = 1, id, gap, ...styleProps }: ListItemLoaderProps) => {
   return (
-    <span id={id}>
+    <ListWrapper gap={gap} id={id}>
       {repeat(
         (index) => (
-          <Wrapper key={index} {...styleProps}>
+          <ItemWrapper key={index} {...styleProps}>
             {children}
-          </Wrapper>
+          </ItemWrapper>
         ),
         count
       )}
-    </span>
+    </ListWrapper>
   )
 }
 
-const Wrapper = styled.div<ListItemLoaderProps>`
+const ListWrapper = styled.span<{ gap?: string }>`
```

Comment:
To prevent the React dom nesting warning:
```suggestion
const ListWrapper = styled.div<{ gap?: string }>`
```